cmd/ocean/account.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 var (
-	accountName     string
+	label           string
+	namespace       string
```

Comment:
Let's revert these changes. An account name can be either a namespace or label.

```suggestion
	accountName           string
```